Dockerfile

```diff
@@ -8,11 +8,21 @@ ENV NODE_OPTIONS="--max-old-space-size=4096"
 
 WORKDIR /app
 COPY frontend/ .
-RUN if [ "$BUILDARCH" != "amd64" ]; then apt-get update && apt-get install -y build-essential python3 --no-install-recommends; fi
-RUN npm install -g pnpm@9 && pnpm install --frozen-lockfile
-RUN pnpm run docker:build
+RUN if [ "$BUILDARCH" != "amd64" ]; then \
+      apt-get update && \
+      apt-get install -y build-essential python3 --no-install-recommends; \
+    fi && \
+    npm install -g pnpm@9 && \
+    pnpm install --frozen-lockfile && \
+    pnpm run docker:build
 
-FROM python:3.11-buster AS backend-build-stage
+FROM rust:1-bookworm AS colibri-build-stage
```

Comment:
what is bookworm?

Reply:
It's my favorite light novel https://en.wikipedia.org/wiki/Ascendance_of_a_Bookworm !!!, no, just kidding :stuck_out_tongue: it's not this bookworm.

So **buster** is **Debian 10**: 

> *(Security updates have been discontinued as of June 30th, 2022)*
- https://www.debian.org/releases/buster/

**bookworm** is **Debian 12**:

> *(full Debian support, until June 10th, 2026, and two years of 
Long Term Support (LTS), until June 30th, 2028)*

- https://www.debian.org/releases/bookworm/

In the docker image context it just means that the docker image uses the specific **Debian** version as it's base. And I wanted to update the build stages because they are outdated.

The final image we ship was safe since **nginx:1.26** is already based on **bookworm**